client/lib/translator-jumpstart/index.js

```diff
@@ -109,8 +106,11 @@ const communityTranslatorJumpstart = {
 			props[ 'data-plural' ] = optionsFromPage.plural;
 		}
 
-		// React.DOM.data returns a frozen object, therefore we make a copy so that we can modify it below
-		const dataElement = Object.assign( {}, React.DOM.data( props, displayedTranslationFromPage ) );
+		// <data> returns a frozen object, therefore we make a copy so that we can modify it below
+		const dataElement = Object.assign(
+			{},
+			<data { ...props }>{ displayedTranslationFromPage }</data>
+		);
```

Comment:
👍 I tried this originally and got a bunch of errors, not realising that these errors had [existed all along](https://github.com/Automattic/wp-calypso/issues/20534#issuecomment-349467563)!

Reply:
Same here, glad that you saw through it Alex

Reply:
FWIW these warnings come from the fact that these props usually get back a string from `translate()` but when the Community Translator is activated, we return an object (the `<data>`) that [properly transform into a string at the DOM level](https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute) ("value: A DOMString containing the value to assign to the attribute. Any non-string value specified is converted automatically into a string."). That's all the message is about.